pkg/kubelet/dockershim/convert.go

```diff
@@ -141,6 +144,9 @@ func toRuntimeAPISandboxState(state string) runtimeApi.PodSandBoxState {
 
 func toRuntimeAPISandbox(c *dockertypes.Container) (*runtimeApi.PodSandbox, error) {
 	state := toRuntimeAPISandboxState(c.Status)
+	if len(c.Names) == 0 {
+		return nil, fmt.Errorf("unexpected empty sandbox name: %+v", c)
```

Comment:
so never restart a reastart:never pod then? I'm not sure what "Dead" means in this context even, something was docker rm'd while it had live mounts?